.pylintrc

```diff
@@ -9,9 +9,10 @@ disable=
     too-many-instance-attributes,
     too-many-public-methods,
     no-else-return,
-    no-self-use,
     duplicate-code,
-    fixme
+    fixme,
+    unspecified-encoding,
```

Comment:
I agree with ignoring `consider-using-f-string` as there isn't really anything wrong with not using it (sometimes shorter). But maybe specifying the encoding is good?

Reply:
`read_text` is what reads bytes and decodes them into text. That is where you can add the `encoding` argument.
e.g.

```python
read_text(encoding='utf-8')
````

The idea behind the recommendation of adding it, is to make it explicit for each use. Rather than relying on some environment specific defaults that may only make sense for specific cases.